pipeline_steps/common.groovy

```diff
@@ -354,6 +354,16 @@ def prepareConfigs(Map args){
 
 def prepareRpcGit(Map args){
   dir("/opt/rpc-openstack"){
+
+    /* if job is triggered by PR, then we need to set RPC_REPO and
+       RPC_BRANCH using the env vars supplied by ghprb.
+    */
+    if ( ghprbPullId != null  ){{
```

Comment:
Double braces are used in JJB jobs to escape the jjb template mechanism. In direct groovy code single braces should be used.